tests/test_wikipedia_linking.py

```
@@ -0,0 +1,258 @@
+        assert summary == ""
+        assert thumbnail is None
+
+# Integration tests (require actual API keys and network access)
+@pytest.mark.integration
+class TestWikipediaLinkingIntegration:
+    """Integration tests for Wikipedia linking (require API access)"""
+    
+    @pytest.mark.skip(reason="Requires OpenAI API key")
+    def test_real_entity_linking(self):
+        """Test actual entity linking with real APIs"""
+        service = WikipediaLinkingService(similarity_threshold=0.3)
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'similarity_threshold' in constructor call

(E1123)